### PR TITLE
Added PyLint spelling private dictionary and fixed typos

### PR DESCRIPTION
**Pull request by joachimmetz**
Added PyLint spelling private dictionary and fixed typos

### CODE REVIEW

File: .pylint-spelling-private-dict [hunk 1]
```diff
@@ -0,0 +1,119 @@
+abspath
```
Comment (Onager) [inline .pylint-spelling-private-dict]: Can we move this file to somewhere else - config/ or data/?
This filename doesn't seem to be required by pylint, I imagine we'll need to edit it fairly regularly, and I don't see the need for it to be in the project root.

File: .pylint-spelling-private-dict [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+diffbase
+distributable
+dmg
+dockerfile
+dockerized
+docopt
+dokan
+dpkg
+elif
+elper
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply .pylint-spelling-private-dict]: fixed

File: .pylint-spelling-private-dict [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+exe
+exensions
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply .pylint-spelling-private-dict]: fixed

File: .pylint-spelling-private-dict [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+boolean
+buildpackage
+cfg
+changelog
+cli
+cls
+compat
+config
+configparser
+configuation
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply .pylint-spelling-private-dict]: fixed

File: .pylint-spelling-private-dict [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+contibution
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply .pylint-spelling-private-dict]: fixed

File: .pylint-spelling-private-dict (hunk 1 above)
Comment (joachimmetz) [reply .pylint-spelling-private-dict]: This will be project dependent I can move it to `config/pylint/` if you think that is a more appropriate location.

**Comment by joachimmetz** [comment]
@Onager PTAL, so I can make the same changes for other log2timeline projects.

**Pull request merged**